src/AppBundle/Twig/FieldOptionsTwigExtension.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace AppBundle\Twig;
+
+use Twig_Extension;
+use Twig_SimpleFunction;
+use eZ\Publish\API\Repository\ContentTypeService;
+
+class FieldOptionsTwigExtension extends Twig_Extension
+{
+    /** @var \eZ\Publish\API\Repository\ContentTypeService */
+    private $contentTypeService;
+
+    /**
+     * @param \eZ\Publish\API\Repository\ContentTypeService $contentTypeService
+     */
+    public function __construct(ContentTypeService $contentTypeService)
+    {
+        $this->contentTypeService = $contentTypeService;
+    }
+
+    /**
+     * Returns extension name.
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return 'ez_field_options';
+    }
+
+    /**
+     * Returns a list of functions to add to the existing list.
+     *
+     * @return array
+     */
+    public function getFunctions()
+    {
+        return [
+            new Twig_SimpleFunction('ez_field_options', [$this, 'showFieldOptions'], [
```

Comment:
I totally understand this twig function. 
IMHO, this should be included in the product as the other functions (https://doc.ez.no/display/EZP/Twig+functions).
What do you think?